### ISSUE TITLE
标签云显示不完整

### ISSUE THREAD

**Issue by ziyunch** [iissnan/hexo-theme-next#1908]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.iissnan.com/).
- [x] And yes, I already searched for current [issues](https://github.com/iissnan/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
The tags page would contain all tags in my site, which are shown in a form of tagcloud. (Total tags number is over 200)
标签页面以标签云形式显示所有标签（标签数>200）

### Actual behavior 
Some tags are not shown in the tagcloud, but we could still retrieve the page for the individual tag by the url (which means the process of tag generation is good).
标签云显示不完整：部分标签未显示于标签云内，但是可以通过输入地址访问（说明标签生成没有问题）

### Steps to reproduce the behavior 
1. Create a page named tags
2. Edit tags page, set page type to tags.
3. Add tags to theme _config.yml:

* Link to demo site with this issue: [demo](http://blindwith.science/tags/)
* Link(s) to source code or any usefull link(s): 

In index.md file in tags folder, the content is:
在tags目录中的index.md文件内容如下：

```
---
title: tags
date: 2016-10-26 23:15:43
type: "tags"
comments: false
---
```

### NexT Information

**NexT Version:**

```
[x] Latest Master branch.
[] Latest Release version.
[] Old version - 
```

**NexT Scheme:**

```
[] All schemes
[] Muse
[] Mist
[] Pisces
[x] Gemini
```

**Comment by ivan-nginx** [comment]
Confirmed. If tags cloud have many tags, pagination is not adding.

**Comment by ivan-nginx** [comment]
@ziyunch ok, i find this and it's actually not a bug and maybe performance option.

For tag cloud generation, [tagcloud](https://hexo.io/docs/helpers.html#tagcloud) ([tagcloud.js](https://github.com/hexojs/hexo/blob/4f190623ebe9c73560b2b50625b17ed35ab9c3eb/lib/plugins/helper/tagcloud.js#L201)) helper used by NexT theme in [page.swig](https://github.com/iissnan/hexo-theme-next/blob/22bbabebd9295688b5f1aabec9abdda9d277f4a7/layout/page.swig#L37) file.

And as we can see, in `tagcloud` default amount of tags — **`40`**
And in NexT `page.swig` value is set to **`200`**:
```swig
{{ tagcloud({min_font: 12, max_font: 30, amount: 200, color: true, start_color: '#ccc', end_color: '#111'}) }}
```
Can u test it for play around with `amount: 200` option? Set to `0` or to `300` and see what happening.
I think need to add option to this amount or add pagination. What u think, how will be better?

But always remember about performance. [Here](https://github.com/hexojs/hexo/issues/1769) is talking about speed generating and `tagcloud` helper figured there. Can u tell me **how many seconds/minutes** your blog generate is spending with **~400** posts and **~200 tags** in NexT?

**Comment by ivan-nginx** [comment]
Added temporary solution in de271c1.

**Comment by ziyunch** [comment]
@ivan-nginx It works now, thanks.

By the way, it took 3.91 s to load and 50 s to generate all the files.

**Comment by ivan-nginx** [comment]
Added cache option to increase generation speed in [NexT v6.0.0](https://github.com/theme-next/hexo-theme-next/releases/tag/v6.0.0).

**Comment by ivan-nginx** [comment]
Fixed in [#662](https://github.com/theme-next/hexo-theme-next/pull/662) pull.
